creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -419,6 +419,25 @@ an IP address configured, which may introduce issues with third-party solutions
 that may rely on it. For example, Istio may not work in this mode.
 ______________________________________________________________________________
 
+______________________________________________________________________________
+*Note:* there is an option to dissable bridge interface with pod network.
+This is done with a designated flag.
+The default value of the flag is 'true', meaning bridge interface is enabled.
+to disable it change flag value to false. For example:
+______________________________________________________________________________
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-config
+  namespace: kube-system
+  labels:
+    kubevirt.io: ""
+data:
+ permitBridgeInterfaceOnPodNetwork: "false"
+______________________________________________________________________________
+*Note:* consider changing the default interface in the config map to masquerade.
```

Comment:
why should I though? I'm still not sure if I'm going to be affected by any issues if I don't use third party solutions like Istio.

Reply:
Thanks @booxter . you are right, that is a bit misleading.
I added some example reasons why one would prefer to not use bridge, and a general concern about to bridge binding to reason the recommendation to use masquerade. 
+  making the explanation about the third party issue and the recommendation to use masquerade in one note, or keep as it is - separate?   
What do you think?

Reply:
I don't really care, both are ok in my view.